Tests/iaas/SCS-Spec.Images.yaml

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
I understand where you come from, but we already provide a set of standard images in the form https://github.com/SovereignCloudStack/standards/blob/main/Tests/iaas/scs-0104-v1-images.yaml.
So IMO, its not that much of a deal to also provide this other file, since its practically is derived from the first one.
And we also don't say that this is the only solution, but its quite a good guidance.

Reply:
The set of standard images you're referring to is merely fixing the image source, so people know what to expect when they encounter a certain name. The spec file, on the other hand, contains quite a lot of specific details that I (as a standards person) have absolutely no expertise about. So I'm not convinced.

Reply:
I agree with Hannes here: to make it easier for a CSP to become compliant, it would be really nice to have a complete, working YAML that can just be fed to the `openstack-image-manager`. At the moment, you'll need to assemble parts of it yourself (I exercised this recently):

- go to https://github.com/osism/openstack-image-manager/tree/main/etc/images
- find and copy ubuntu and debian files
- filter unnecessary things from the ubuntu files (e.g., versions 18.04 and older, "Minimal" variants)
- for the CAPI images, it is even more difficult, because the `kubernetes.yml` in that directory is not up-to-date
  - in the OSISM stack, the `osism` cli tool is used for this purpose (`osism manage image clusterapi` as mentioned [here](https://github.com/SovereignCloudStack/standards/issues/569#issuecomment-2063919012))
  - it creates an up-to-date YAML on the fly and feeds it to the image manager (https://github.com/osism/python-osism/blob/main/osism/commands/manage.py)

However, I disagree with the file name choice: it should reflect that it is meant for v1 of scs-0104 and to be fed to the image manager.

Reply:
Yes, people will have to do this. However, we can't keep this file up to date, which is what OSISM (semi-) automatically does in their repo. That we used the `versions` wrong is just the evidence that we shouldn't meddle in this.